src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml

```diff
@@ -1,18 +1,28 @@
 ﻿@model VolleyManagement.UI.Areas.Mvc.ViewModels.GameResults.GameResultViewModel
 
+<link rel="stylesheet" href="~/Content/Site.css" />
```

Comment:
What is this for?

Reply:
To connect styles

Reply:
Weren't they already working?

Reply:
No, not work.

Reply:
Hm... Indeed it is not working. Remove it from here and add Site.css to the "commoncss" bundle.

Reply:
"commoncss" bundle???

Reply:
https://github.com/VolleyManagement/volley-management/blob/master/src/VolleyManagement.Backend/VolleyManagement.UI/App_Start/BundleConfig.cs

Reply:
Thanks)